src/Toolchain/ToolchainProvider.ts

```diff
@@ -57,25 +64,29 @@ export class ToolchainProvider implements vscode.TreeDataProvider<ToolchainNode>
     if (element) {
       return Promise.resolve(this.getNode(element));
     } else {
-      const nodes = Object.keys(gToolchainEnvMap)
-                        .map((backend) => this.toToolchainNode(NodeType.backend, backend));
-      return Promise.resolve(nodes);
+      return Promise.resolve(this.getNode(undefined));
```

Comment:
Not `reject()`?

Reply:
When the toolchain view is loaded at the first, the element is undefined and it should create the element by itself. So it calls `getNode()`.

Reply:
Hmm... The reason why I asked is...
```js
    if (element) {
      return Promise.resolve(this.getNode(element)); 
    } else {
      return Promise.resolve(this.getNode(undefined));
    }
// is same to 
return Promise.resolve(this.getNode(element)); // it's because if element is undefined, `undefined` is passed.
```

Reply:
applied :)